indico/modules/search/result_schemas.py

```diff
@@ -68,22 +71,23 @@ def _get_url(self, data):
 
 
 class LocationResultSchema(mm.Schema):
-    venue_name = fields.String(required=True)
-    room_name = fields.String(required=True)
-    address = fields.String(required=True)
+    venue_name = fields.String(required=True)  #: The venue name
+    room_name = fields.String(required=True)  #: The room name
+    address = fields.String(required=True)  #: The address
 
 
 class EventResultSchema(ResultSchemaBase):
-    type = EnumField(SearchTarget, validate=require_search_target(SearchTarget.event))
-    event_id = fields.Int(required=True)
-    title = fields.String(required=True)
-    description = fields.String(required=True)
-    event_type = EnumField(EventType, required=True)
-    start_dt = fields.DateTime(required=True)
-    end_dt = fields.DateTime(required=True)
-    persons = fields.List(fields.Nested(PersonSchema), required=True)
-    location = fields.Nested(LocationResultSchema, required=True)
-    highlight = fields.Nested(HighlightSchema, missing={})
+    #: The record type
+    type: SearchTarget = EnumField(SearchTarget, validate=require_search_target(SearchTarget.event))
+    event_id = fields.Int(required=True)  #: The event id
```

Comment:
I'd still put all of them before the line  - otherwise it's a bit inconsistent, and every doc comment starting at a different horizontal position looks rather strange IMO